### ISSUE TITLE
Accessing and modifying localized versions of pages and web parts with the Graph REST API

### ISSUE THREAD

**Issue by Hippu** [SharePoint/sp-dev-docs#9866]
### What type of issue is this?

Question

### What SharePoint development model, framework, SDK or API is this about?

SharePoint REST API

### Target SharePoint environment

SharePoint Online

### What browser(s) / client(s) have you tested

- [ ] 💥 Internet Explorer
- [ ] 💥 Microsoft Edge
- [ ] 💥 Google Chrome
- [ ] 💥 FireFox
- [ ] 💥 Safari
- [ ] mobile (iOS/iPadOS)
- [ ] mobile (Android)
- [X] not applicable
- [ ] other (enter in the "Additional environment details" area below)

### Additional environment details

I tried setting the Accept-Language header with the MS Graph Python SDK and the Graph Explorer 

### Issue description

Hi. Is it possible to access and modify localized version of web parts with the Graph APIs?
I couldn't find any documentation on how to do that. I also tried setting the Accept-Language header, but that didn't seem to work.

The microsoft.graph.sitePage/webParts resource only returns the English version when I want to get the Finnish version.

Thanks in advance!

**Comment by Ashlesha-MSFT** [comment]
Hello @Hippu,
Thank you for bringing this issue to our attention. We will look into it and get back to you shortly.

**Comment by Ashlesha-MSFT** [comment]
@Hippu,
You're correct — currently, the Microsoft Graph API (microsoft.graph.sitePage/webParts) does not provide localized versions of web parts directly, and setting the Accept-Language header doesn’t retrieve localized variants of the content. 
For SPFx developers, localization is handled through resource files at the development level. You can refer to the official documentation here:  https://learn.microsoft.com/en-us/sharepoint/dev/spfx/web-parts/guidance/localize-web-parts

**Comment by Ashlesha-MSFT** [comment]
We are closing this issue for now. If the problem persists, feel free to reopen it or open a new one.
